recipe/meta.yaml

```
@@ -24,7 +24,7 @@ requirements:
   run:
     - python >=3.6
     - conda-build >=3.24,<3.26
-    - mamba >=1.0,<=1.4.2
+    - mamba >=1.4.5,<=1.4.9
```

Can we try the most recent release?

```suggestion
    - mamba >=1.5.0
```